sh/configure_env.sh

```diff
@@ -0,0 +1,106 @@
+#
+# Author: Philip Deegan
+# Email : philip.deegan@polytechnique.edu 
+# Date  : 21 - September - 2017
+#
+# This script is to be included to setup variables to build tick
+#   It is done so to be cross platform and heterogeneous
+#
+# Variables
+#  $PY = "python" binary
+#  $PCONF = "python-config" binary
+#  $LDARGS=Linking arguments
+#  $LIB_POSTFIX= python library format 
+#  $LIB_POSTEXT= system library file extention
+#  $PROFILES= array of mkn profiles to build
+# 
+######################################################################
+
+set -e
+
+[ -z "$PY" ] && which python3 &> /dev/null && PY="python3"
+[ -z "$PY" ] && which python &> /dev/null && PY="python"
+[ -z "$PY" ] && echo "python or python3 not on path, exiting with error" && exit 1
+
+[ -z "$PCONF"  ] && which python3-config &> /dev/null && PCONF="python3-config"
+[ -z "$PCONF"  ] && which python-config &> /dev/null && PCONF="python-config"
+[ -z "$PCONF"  ] && echo "python-config or python3-config not found on PATH, exiting with error" && exit 1;
+
+[ -z "$PINC" ] && for i in $($PCONF --includes); do PINC="${PINC}${i:2}:"; done
+[ -z "$PNIC" ] && PNIC=$($PY -c "import numpy as np; print(np.get_include())");
+
+LDARGS="$($PCONF --ldflags)"
```

Comment:
Please explain why we need this sed :)

Reply:
In the file or here? 

sed is used there to remove parts of the string, if for instance we wish to override them with other values.

Reply:
Yes I know since you were nect to me when you added it but I think it deserves a comment in the code, just like the above lines

Reply:
Comments added